src/note.h

```diff
@@ -44,35 +43,31 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  * */
 
 enum ecolor {
-   COLOR_ERROR    = -1, /* error parser                        */
-   COLOR_OPTIONAL = 0,  /* the user did not indicate the color */
-   COLOR_OK       = 1,  /* the user indicate the color         */
+    COLOR_ERROR = -1, /* error parser                        */
```

Comment:
I think we should stick with what the style indicates: _Enum members should use InterCaps with an initial capital letter._
[As you have done before](https://github.com/resurrecting-open-source-projects/scrot/pull/125/files#diff-5dcac73d8825241270f8f0d94de4dd007e442c6ca9c079eeb84f2971d2ca04b8R90)
```C
    enum {
        Style = 0,
        Width,
        Color,
        Opacity,
        Mode
    };
```

Reply:
Fixed in 214c995